src/docs-assembler/navigation.yml

```diff
@@ -467,23 +483,28 @@ toc:
                 # Children include the entire AsciiDoc book
 
       # Data analysis
-      # 🔜 https://github.com/elastic/docs-content/blob/prepare-for-assembler/reference/data-analysis/toc.yml
+      # ✅ https://github.com/elastic/docs-content/blob/main/reference/data-analysis/toc.yml
       - toc: docs-content://reference/data-analysis
         path_prefix: reference/data-analysis
         # Children include: Supplied configurations, Function reference,
         # Metrics reference, Canvas function reference
         children:
-          # 📝 TO DO: Directory depth in elasticsearch repo: would require toc 2 levels down
-          # 🔜 https://github.com/colleenmcginnis/elasticsearch/blob/docs-assembler-prep/docs/reference/text-analysis/toc.yml
+
+          # 📝 TO DO: Delete this item
+          # This should be replaced by `elasticsearch://reference/text-analysis` and
+          # `elasticsearch://reference/aggregations` below.
+          - toc: elasticsearch://reference/data-analysis
+            path_prefix: reference/elasticsearch/data-analysis
+
+          # ✅ https://github.com/elastic/elasticsearch/blob/main/docs/reference/text-analysis/toc.yml
           - toc: elasticsearch://reference/text-analysis
             path_prefix: reference/text-analysis
-          # 📝 TO DO: Directory depth in elasticsearch repo: would require toc 2 levels down
-          # 🔜 https://github.com/colleenmcginnis/elasticsearch/blob/docs-assembler-prep/docs/reference/aggregations/toc.yml
+          # ✅ https://github.com/elastic/elasticsearch/blob/main/docs/reference/aggregations/toc.yml
           - toc: elasticsearch://reference/aggregations
             path_prefix: reference/aggregations
```

Comment:
I would expect we'd be able to remove `elasticsearch://reference/data-analysis` in favor of  `elasticsearch://reference/text-analysis` and `elasticsearch://reference/aggregations` below now that https://github.com/elastic/elasticsearch/pull/125118 was merged, but at least when I run `build-all` locally I get errors if I delete this.

Reply:
Hey colleen you might need to delete your `.artifacts/checkouts` folder and call `clone-all` again. 

Alternatively you **should** be able to call `clone-all` without deleting to update all but I have not tested if it works in all scenario's yet.